src/tests/test_affiliate/WidgetsTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Organic;
+
+use Exception;
+use PHPUnit\Framework\TestCase;
+
+include( __DIR__ . '/../SeleniumBrowser.php' );
+
+define( "Organic\WP_VERSION", getenv( 'WP_VERSION' ) ?? '' );
+
+class WidgetsTest extends TestCase {
+
+    /**
+     * Tests for custom widgets.
+     * Note that these tests will fail for non-Gutenberg WordPress (< version 5).
+     */
+
+    function wordPressVersionTooLow() : bool {
```

Comment:
The format in this file is mixed up, but I'm not sure exactly what we want to do since we tend to use snake_case for our own code but other PHP libraries use camelCase.

Reply:
Actually we do use camelCase in the Organic WP plugin, the only snake_case code is methods provided by Wordpress itself